repose-aggregator/src/docs/asciidoc/ver-9-upgrade-notes.adoc

```diff
@@ -1,4 +1,25 @@
 = Upgrade to Version 9 from Version 8 Notes
 
+== Core
```

Comment:
I'm not certain this section is necessary. We don't normally go into detail about changes in internal APIs. I feel like this is aimed at people who just care about configuring. I'm not against making a specific section for integrators though where we give details like this and we do a guide everytime we bump the first or second digit.

Reply:
I was wondering about that myself. I asked myself WWMD (what would Mario do) and decided to just type everything up since it's easy enough to move or remove.

I'm cool with sticking this in an integrator section.

Reply:
Yeah, lets go ahead and make a specific section for that